server/stycoon/history.go

```diff
@@ -0,0 +1,81 @@
+package stycoon
+
+import (
+	"database/sql"
+	"encoding/json"
+)
+
+func (game *Game) reportStaticData() error {
+	_, err := game.db.Exec("insert into d_history (`season`, `tick`, `static-data`) values (?, ?, ?)", game.Tick.Season, -1, game.SerializedStaticData)
+	return err
+}
+
+func (game *Game) report() error {
+	// TODO report report
+	reportUserId := int64(-1)
+	data, err := game.GetData(&reportUserId)
+	if err != nil {
+		return err
+	}
+	serializedData, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+	_, err = game.db.Exec("insert into d_history (`season`, `tick`, `data`) values (?, ?, ?)", game.Tick.Season, game.Tick.Tick, serializedData)
```

Comment:
season and tick is a primary key. multiple inserts will fail. it must be inserted all at once

Reply:
I count with that. It also depends on how we are going to store reports.